lib/puma/events.rb

```diff
@@ -1,52 +1,23 @@
 # frozen_string_literal: true
 
-require "puma/null_io"
-require 'puma/error_logger'
-require 'stringio'
-
 module Puma
-  # The default implement of an event sink object used by Server
-  # for when certain kinds of events occur in the life of the server.
-  #
-  # The methods available are the events that the Server fires.
-  #
-  class Events
-    class DefaultFormatter
-      def call(str)
-        str
-      end
-    end
-
-    class PidFormatter
-      def call(str)
-        "[#{$$}] #{str}"
-      end
-    end
-
-    # Create an Events object that prints to +stdout+ and +stderr+.
-    #
-    def initialize(stdout, stderr)
-      @formatter = DefaultFormatter.new
-      @stdout = stdout
-      @stderr = stderr
 
-      @debug = ENV.key? 'PUMA_DEBUG'
-      @error_logger = ErrorLogger.new(@stderr)
+  # This is an event sink used by `Puma::Server` to handle
+  # lifecycle events such as :on_booted, :on_restart, and :on_stopped.
+  # Using `Puma::DSL` it is possible to register callback hooks
+  # for each event type.
+  class Events
 
+    def initialize
```

Comment:
Removing the arguments here has blown up a few of the tests

Reply:
@nateberkopec I'm aware that this has broken and will fix it. May I get some comments on whether or not at a high-level the change in this PR is acceptable/desirable for Puma?